get_age.py

```diff
@@ -1,21 +1,43 @@
 import sys
 import datetime
+import os
+import time
+import platform
 
-year = int(input("What year were you born? "))
-month = int(input("What month were you born? "))
-day = int(input("What day were you born? "))
+def clear():
+	if platform.system() == "Linux":
+		os.system("clear")
+	elif platform.system() == "Windows":
+		os.system("cls")
+	else:
+		pass
 
-yes_no = input("So you were born on %02d/%02d/%04d. Is that correct? [yes/no]" % (day, month, year))
-if yes_no.lower() == "no":
-    print("Too bad!")
-else:
-    print("Excellent!")
-
-    today = datetime.date.today()
-    age = today.year - year
-    if month > today.month:
-        age -= 1
-    elif month == today.month and day > today.day:
-        age -= 1
-
-    print("You are", age, "years old")
\ No newline at end of file
+def ask():
+    year = int(input("What year were you born? "))
+    month = int(input("What month were you born? "))
+    day = int(input("What day were you born? "))
+    yes_no = input("So you were born on %02d/%02d/%04d. Is that correct? [yes/no]" % (day, month, year))
+    if yes_no.lower() == "no":
```

Comment:
Really the "Ask your date of birth" functionality ought to be broken out into its own function and either that or this piece of code can loop until it gets a satisfactory "Yes".

Getting ask to call ask() as you're doing is a nice idea and it will work -- it's called recursion -- but it's not really the right tool for the job here. It's a bit complicated to explain why, but this would be better with an iteration (while <whatever>: ...).

Refactoring this would also make it more obvious where the calls to clear() should sit: at the moment they're a bit haphazard.